go/oasis-node/cmd/debug/txsource/workload/governance.go

```diff
@@ -143,7 +143,7 @@ func (g *governanceWorkload) doUpgradeProposal() error {
 	maxUpgradeEpoch := minUpgradeEpoch + int64(3*g.parameters.UpgradeMinEpochDiff)
 	// [minUpgradeEpoch, maxUpgradeEpoch]
 	upgradeEpoch := beacon.EpochTime(g.rng.Int63n(maxUpgradeEpoch-minUpgradeEpoch+1) + minUpgradeEpoch)
-	nameSuffix := make([]byte, 20)
+	nameSuffix := make([]byte, 16)
```

Comment:
This is probably still too high as it gets hex encoded.

Reply:
Oh, of course, 16 byte suffix translates to 32 characters in hex encoding.

I've shortened it to 8 bytes.